tests/test_cases/input/escline_node.kdl

```diff
@@ -1,2 +1,3 @@
 node1
+\
```

Comment:
Based on the filename I figured this test was missing something...

Reply:
This is allowed per spec, but I'm surprised by that.

`line-space` doesn't contain `escline`, only `node-space` has `escline`
All whitespace between nodes is `line-space`, so on first sight this test would be invalid, but&hellip;
`base-node` starts with `slashdash? type? node-space* string` which means this is valid

```kdl
node1
\
node2
```

but this isn't

```kdl
(lorem)node1
\
(lorem)node2
```

I personally always read (and implemented) the start of `base-node` as `slashdash? (type node-space*)? string`, as I interpreted the node-space to be meant as space between the type and name.

Reply:
...I noticed this recently in kdl-rs too and it made me wonder whether `escline` _should_ be part of `line-space`.

Reply:
what in the spec allows this, btw?

Reply:
Ah I see:
```
(type) \
node
```

is allowed and `(type)` is optional, therefore
```
\
node
```
is allowed.

Quite separately,
```
/- \
(type)node
```
and even
```
/-
\
(type)node
```
are allowed because `/-` can always be followed by `node-space`, even when not in a node.

IMHO, this is too confusing and `escline` belongs in `line-space`.